src/coreclr/jit/codegenxarch.cpp

```diff
@@ -489,7 +489,8 @@ void CodeGen::genSetRegToConst(regNumber targetReg, var_types targetType, GenTre
             emitter* emit = GetEmitter();
             emitAttr attr = emitTypeSize(targetType);
 
-            if (vecCon->IsAllBitsSet())
+            // TODO-XARCH-AVX512: Remove the TYP_SIMD64 check once AllBitsSet for TYP_SIMD64 gets codegen support
```

Comment:
Will it help if I put the AllBitSet PR up for review today? I have it ready I have been looking to add a few other s to it. But just make sense to add it before to prevent this part

Reply:
Probably doesn't matter too much either way. It should be a simple fix whether this or the AllBitsSet PR is merged first.

I actually intended `AllBitsSet` to be working with the PR that added TYP_SIMD64 support to GenTreeVecCon, but it got missed since a test didn't exist yet.